app/domain/authentication/authn_jwt/jwt_orchestrator.rb

```diff
@@ -0,0 +1,21 @@
+require 'command_class'
+
+class JWTOrchestrator
+  module AuthnJwt
+
+    Authenticate ||= CommandClass.new(
```

Comment:
Change Authenticate to JWTOrchestrator

Reply:
what does this class do? Does it orchestrate a JWT? Does it do it in one command? if so the name should be `OrchestrateJWT` to tell the action it does

Reply:
the name of a command class should be self-explanatory. 
I must say that i don't understand what `OrchestrateJWT` is.
I would start by writing a description saying what this class does and how you should use it. The name should pop from that.

Reply:
Basically this class is supposed to orchestrate between different vendors configurations and authenticate according to each configuration 

Maybe we should call it OrchestrateJwtVendorConfiguration? @orenbm

Reply:
does it create a Conjur access token at the end of its process?

Reply:
As far as I know, it is

@nessiLahav can you confirm?